cypress/e2e/marc-authority/marc-authority-edit-010.cy.js

```diff
@@ -0,0 +1,107 @@
+import getRandomPostfix from '../../support/utils/stringTools';
+import { DevTeams, Permissions, TestTypes } from '../../support/dictionary';
+import TopMenu from '../../support/fragments/topMenu';
+import Users from '../../support/fragments/users/users';
+import MarcAuthorities from '../../support/fragments/marcAuthority/marcAuthorities';
+import MarcAuthority from '../../support/fragments/marcAuthority/marcAuthority';
+import DataImport from '../../support/fragments/data_import/dataImport';
+import JobProfiles from '../../support/fragments/data_import/job_profiles/jobProfiles';
+import Logs from '../../support/fragments/data_import/logs/logs';
+import QuickMarcEditor from '../../support/fragments/quickMarcEditor';
+
+describe('Edit Authority record', () => {
+  const testData = {
+    authority: {
+      searchInput: 'DiCaprio, Leonardo',
+      searchOption: 'Keyword',
+    },
+    editedField: {
+      tag: '035',
+      content: 'edited 035',
+    },
+    fields010: [
+      { rowIndex: 4, tag: '010', content: '$a n  94000330' },
+      { rowIndex: 5, tag: '010', content: '$a n  94000339' },
+    ],
+  };
+  const subfieldPrefix = '$a';
+  const authorityPostfix = '?authRefType=Authorized&heading';
+  const jobProfileToRun = 'Default - Create SRS MARC Authority';
+  const marcFiles = [
+    {
+      marc: 'marcAuthFileForC375127.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      numOfRecords: 1,
+    },
+  ];
+  const createdAuthorityIDs = [];
+
+  before('Upload files', () => {
+    cy.loginAsAdmin({ path: TopMenu.dataImportPath, waiter: DataImport.waitLoading }).then(() => {
+      marcFiles.forEach((marcFile) => {
+        DataImport.verifyUploadState();
+        DataImport.uploadFile(marcFile.marc, marcFile.fileName);
```

Comment:
Please, use the modern method uploadFileAndRetry
Refactor this block according new approach to import files
See examples with uploadFileAndRetry